documentation/bolt_installing_modules.md

```diff
@@ -329,44 +329,136 @@ module-install:
-       you want to install. If the modules have dependencies, list those as
-       well. For example:
-       ```puppet
-       # Modules from the Puppet Forge.
-       mod 'puppetlabs-apache', '4.1.0'
-       mod 'puppetlabs-postgresql', '5.12.0'
-       mod 'puppetlabs-puppet_conf', '0.3.0'
-
-       # Modules from a Git repository.
-       mod 'puppetlabs-haproxy', git: 'https://github.com/puppetlabs/puppetlabs-haproxy.git', ref: 'master'
-       ```   
-   1. Run the `bolt puppetfile install` command. Bolt installs modules to the
-      first directory in the `modulepath` setting. By default, this is the
-      `modules/` subdirectory inside the Bolt project directory. To override
-      this location, update the `modulepath` setting in your [project
-      configuration file](bolt_project_reference.md).
+## Manually manage a project's modules
+
+If Bolt can't resolve your module dependencies, you must manage your project's
+Puppetfile manually and use Bolt to install the modules listed in the Puppetfile
+without resolving dependencies. The process for manually managing your modules
+uses the new `module` subcommand, and replaces the now deprecated `puppetfile`
+subcommand.
+
+The most common scenario where Bolt can't resolve module dependencies is when
+a project includes git modules that are in a repository other than a public
+GitHub repository. If your project includes this type of module, you must
+manually manage your project's Puppetfile.
+
+To manually manage a project's Puppetfile and install modules without resolving
+dependencies, follow these steps:
+
+1. Set the `modules` configuration option to an empty array. For example:
+
+   ```yaml
+   # bolt-project.yaml
+   name: myproject
+   modules: []
+   ```
+
+1. Create a file named `Puppetfile` in your project directory and add the
+   modules you want to install, including each module's dependencies, to
+   the Puppetfile. For example:
+
+   ```ruby
+   # Modules from a private git repository
+   mod 'private-module', git: 'https://github.com/bolt-user/private-module.git', ref: 'main'
+
+   # Modules from an alternate Forge
+   mod 'puppetlabs/apache', '5.7.0'
+   mod 'puppetlabs/stdlib', '6.5.0'
+   mod 'puppetlabs/concat', '6.3.0'
+   mod 'puppetlabs/translate', '2.2.0'
+   ```
+
+1. Install the modules in the Puppetfile without resolving dependencies:
+
+   _\*nix shell command_
+
+   ```shell
+   bolt module install --no-resolve
+   ```
+
+   _PowerShell cmdlet_
+
+   ```powershell
+   Install-BoltModule -NoResolve
+   ```
+
+Bolt installs modules in the Puppetfile to the `.modules/` directory in your
+project directory. If you need to add or remove modules to the project, update
+your Puppetfile and run the above command again.
+
+## Migrate a project to use module management
+
+If you created a Bolt project before module dependency management was
+introduced, you can use Bolt's built-in migration tool to [update your
```

Comment:
I feel like this should be in bold maybe? Or otherwise emphasized? It feels easy to skip over.

Reply:
It should be fairly clear in the output because it's right under a H2. Let's see if people have issues with it and revisit.